test/functional/fixtures/runner/uncaught-js-errors/uncaught-js-errors-test.js

```diff
@@ -0,0 +1,73 @@
+var expect    = require('chai').expect;
+var errorType = 'Uncaught JavaScript error Uncaught Error:';
+
+describe('Uncaught js errors.', function () {
+    it('Should fail when are no handler', function () {

```

Comment:
I suppose we should specify what the handler it is not clear now. 
I propose:
`Should fail when there are no onerror handler`
`Should fail when the onerror handler returns undefined`
...